### ISSUE TITLE
[BUG] Error in the dipole 

### ISSUE THREAD

**Issue by PabloAMC** [PennyLaneAI/pennylane#4976]
### Expected behavior

There are a couple of errors in the example of https://docs.pennylane.ai/en/stable/code/api/pennylane.qchem.dipole_of.html
I think it should say
```
symbols = ["H", "H", "H"]
coordinates = np.array([0.028, 0.054, 0.0, 0.986, 1.610, 0.0, 1.855, 0.002, 0.0])
dipole_obs = dipole_of(symbols, coordinates, charge=1)
print(dipole_obs)
```

### Actual behavior

It currently says
```
symbols = ["H", "H", "H"]
coordinates = np.array([0.028, 0.054, 0.0, 0.986, 1.610, 0.0, 1.855, 0.002, 0.0])
dip_obs = dipole(symbols, coordinates, charge=1)
print(dipole_obs)
```

### Additional information

Apart from that, it may be good to pass the coordinates as a 2d array, though the current form is also how it is formatted when creating a molecular Hamiltonian. And perhaps it should be possible to obtain a single operator, rather than just $D_x$, $D_y$ and $D_z$ individually. [`dipole_moment`](https://docs.pennylane.ai/en/stable/code/api/pennylane.qchem.dipole_moment.html) and [`dipole_integrals`](https://docs.pennylane.ai/en/stable/code/api/pennylane.qchem.dipole_integrals.html) seem related but it is unclear how to obtain the `alpha` array values, from a basis set.

### System information

```shell
Does not apply.
```

### Existing GitHub issues

- [X] I have searched existing GitHub issues to make sure the issue does not already exist.

**Comment by trbromley** [comment]
Thanks @PabloAMC!

We can quickly fix the typo, but if you are up for it feel free to make a pull request yourself. This can be done easily by going [here](https://github.com/PennyLaneAI/pennylane/blob/v0.34.0-rc0/pennylane/qchem/openfermion_obs.py#L549) and clicking the edit button in the top right. You can commit changes to a new branch and then make a pull request against the `v0.34.0-rc0` branch.

CC @soranjh

**Comment by soranjh** [comment]
Correcting the typo in the doctring, `dipole` to `dipole_of`, is fine but changing the output format will, very likely, require changing the datasets and therefore should be considered carefully.

> There are a couple of errors

Other than the typo in the docstring example, what other errors are there?

**Comment by PabloAMC** [comment]
I have created it, but am not sure if it is sufficiently important to include it in the `doc/releases/changelog-dev.md`.

**Comment by isaacdevlugt** [comment]
Hey @PabloAMC! Apologies that your PR got closed. There were some hiccups with Github and us releasing a new version of PennyLane, so closing your PR was just an unfortunate accident. 

I've re-opened it here: https://github.com/PennyLaneAI/pennylane/pull/5042

**Comment by PabloAMC** [comment]
Finally closed in #5036 , closing the issue too.
